### DISCUSSION TITLE
Hello I'm meeting this error whlie "NewStream(ctx, info.ID, protocolID)"

### DISCUSSION THREAD

**Discussion by WHOLETTHEDOG-OUT** [libp2p/go-libp2p#3055] [Q&A] [answered]
host.Connect(ctx, *info) is ok;

but 

NewStream(ctx, info.ID, protocolID) meets error: 
failed to open stream: context deadline exceeded

**Comment by hsanjuan** [comment, marked as answer]
Your context expired.
